### PR TITLE
Bug 1853614: Release 4.5 update terraform provider aws 2.67.0

### PR DESCRIPTION
**Pull request by jhixson74**
Update terraform-provider-aws to version 2.67.0 and terraform-plugin-sdk to 1.14.0 to fix https://bugzilla.redhat.com/show_bug.cgi?id=1853614 .

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -115,8 +115,7 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200413201024-c6e8c9b6eb9a // Pin API
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240 // Pin client-go
 	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20200130220348-e5685c0cf530 // Pin MCO so it doesn't get downgraded
-	github.com/ovirt/go-ovirt => github.com/ovirt/go-ovirt v0.0.0-20200320082526-4e97a11ff083
```
Comment (abhinavdahiya) [inline go.mod:139]: i think we should not remove the ovirt replace.

Comment (jhixson74) [reply go.mod:139]: I've updated to include it. I'll push the update when the CI finishes.

**Comment by sdodson** [comment]
@jhixson74 @abhinavdahiya Does this have the follow up fixes for Azure where we regressed there?

**Comment by jhixson74** [comment]
> @jhixson74 @abhinavdahiya Does this have the follow up fixes for Azure where we regressed there?

@sdodson , yes, the terraform provider for Azure has been bumped in this as well.

**Pull request merged**